src/test/java/test/ignore/IgnoreTest.java

```diff
@@ -0,0 +1,60 @@
+package test.ignore;
+
+import org.testng.ITestNGListener;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import test.InvokedMethodListener;
+import test.SimpleBaseTest;
+import test.ignore.ignorePackage.IgnorePackageSample;
+import test.ignore.ignorePackage.subPackage.SubPackageSample;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class IgnoreTest extends SimpleBaseTest {
+
+    @Test
+    public void ignore_class_should_not_run_tests() {
+        TestNG tng = create(IgnoreClassSample.class);
+        InvokedMethodListener listener = new InvokedMethodListener();
+        tng.addListener((ITestNGListener) listener);
+        tng.run();
+
+        // assertThat(listener.getInvokedMethods()).isEmpty();
```

Comment:
If this assertion is not required, can we remove it off ?

Reply:
I don't know. It is the expected assertion but it is not possible by design for the moment.